Integrity test for new directories failing in Wazuh 3.12

## Description

The integrity test for new directories is failing in Wazuh 3.12. Here is the output of the test:
```
=========================================================================================== test session starts ============================================================================================
platform linux -- Python 3.6.8, pytest-5.4.1, py-1.8.1, pluggy-0.13.1
rootdir: /home/vagrant/qa/tests/integration, inifile: pytest.ini
plugins: metadata-1.8.0, html-2.0.1, testinfra-5.0.0
collected 2 items

test_fim/test_basic_usage/test_basic_usage_new_dirs.py .F                                                                                                                                            [100%]

================================================================================================= FAILURES =================================================================================================
__________________________________________________________________________ test_new_directory[get_configuration1-tags_to_apply0] ___________________________________________________________________________

tags_to_apply = {'ossec_conf'}
get_configuration = {'apply_to_modules': ['test_basic_usage_new_dirs'], 'metadata': {'fim_mode': 'whodata', 'module_name': 'test_basic_usa...'no'}}, {'directories': {'attributes': [...], 'value': '/testdir1'}}], 'section': 'syscheck'}], 'tags': ['ossec_conf']}
configure_environment = None, restart_syscheckd = None, wait_for_initial_scan = None

    @pytest.mark.parametrize('tags_to_apply', [
        {'ossec_conf'}
    ])
    def test_new_directory(tags_to_apply, get_configuration, configure_environment, restart_syscheckd,
                           wait_for_initial_scan):
        """
        Check that a new monitored directory generates events after the next scheduled scan.

        This test performs the following steps:
        - Monitor a directory that does not exist.
        - Create the directory with files inside. Check that this does not produce events in ossec.log.
        - Move time forward to the next scheduled scan.
        - Check that now creating files within the directory do generate events.

        Parameters
        ----------
        tags_to_apply : set
            Run test if matches with a configuration identifier, skip otherwise
        """
        check_apply_test(tags_to_apply, get_configuration['tags'])

        if sys.platform != 'win32':
            # Create the monitored directory with files and check that events are not raised
            regular_file_cud(directory_str, wazuh_log_monitor, file_list=['file1', 'file2', 'file3'],
                             min_timeout=global_parameters.default_timeout, triggers_event=False)

            # Travel to the future to start next scheduled scan
            check_time_travel(True)
            detect_initial_scan(wazuh_log_monitor)
        else:
            # Wait for syscheck to realize the directories don't exist
            wazuh_log_monitor.start(timeout=10, callback=callback_non_existing_monitored_dir,
                                    error_message='Monitoring discarded message not found')
            os.makedirs(directory_str, exist_ok=True, mode=0o777)
            time.sleep(windows_audit_interval+0.5)

        # Assert that events of new CUD actions are raised after next scheduled scan
        regular_file_cud(directory_str, wazuh_log_monitor, file_list=['file4', 'file5', 'file6'],
>                        min_timeout=global_parameters.default_timeout, triggers_event=True)

test_fim/test_basic_usage/test_basic_usage_new_dirs.py:100:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/local/lib/python3.6/site-packages/wazuh_testing-3.11.0-py3.6.egg/wazuh_testing/fim.py:1197: in regular_file_cud
    event_checker.fetch_and_check('added', min_timeout=min_timeout, triggers_event=triggers_event)
/usr/local/lib/python3.6/site-packages/wazuh_testing-3.11.0-py3.6.egg/wazuh_testing/fim.py:928: in fetch_and_check
    self.events = self.fetch_events(min_timeout, triggers_event, extra_timeout, error_message=error_msg)
/usr/local/lib/python3.6/site-packages/wazuh_testing-3.11.0-py3.6.egg/wazuh_testing/fim.py:981: in fetch_events
    error_message=error_message).result()
/usr/local/lib/python3.6/site-packages/wazuh_testing-3.11.0-py3.6.egg/wazuh_testing/tools/monitoring.py:187: in start
    error_message=error_message).result()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <wazuh_testing.tools.monitoring.QueueMonitor object at 0x7fc680f9e9b0>, timeout = 5, callback = <function callback_detect_event at 0x7fc67ee562f0>, accum_results = 3, update_position = True
timeout_extra = 0, error_message = "TimeoutError was raised because 3 'added' events were expected for ['/testdir1/file4', '/testdir1/file5', '/testdir1/file6'] but were not detected."

    def start(self, timeout=-1, callback=_callback_default, accum_results=1, update_position=True, timeout_extra=0,
              error_message=''):
        """Start the queue monitoring until the stop method is called."""
        if not self._continue:
            self._continue = True
            self._abort = False

            while self._continue:
                if self._abort:
                    self.stop()
                    if error_message:
                        logger.error(error_message)
                        logger.error(f"Results accumulated: "
                                     f"{len(self._result) if isinstance(self._result, list) else 0}")
                        logger.error(f"Results expected: {accum_results}")
>                   raise TimeoutError()
E                   TimeoutError

/usr/local/lib/python3.6/site-packages/wazuh_testing-3.11.0-py3.6.egg/wazuh_testing/tools/monitoring.py:405: TimeoutError
------------------------------------------------------------------------------------------ Captured stdout setup -------------------------------------------------------------------------------------------
2020-04-30T14:15:18.1588256118
------------------------------------------------------------------------------------------ Captured stderr setup -------------------------------------------------------------------------------------------
2020/04/30 14:15:19 ossec-syscheckd[18885] main.c:100 at main(): DEBUG: Starting ...
2020/04/30 14:15:19 ossec-syscheckd[18885] config.c:68 at Read_Syscheck_Config(): DEBUG: (6287): Reading configuration file: '/var/ossec/etc/ossec.conf'
2020/04/30 14:15:19 rootcheck[18885] rootcheck.c:197 at rootcheck_init(): DEBUG: Starting ...
------------------------------------------------------------------------------------------- Captured stdout call -------------------------------------------------------------------------------------------
2020-05-01T03:15:42.1588302942
------------------------------------------------------------------------------------------- Captured stderr call -------------------------------------------------------------------------------------------
...
-------------------------------------------------------------------------------------------- Captured log call ---------------------------------------------------------------------------------------------
...
----------------------------------------------------------------------------------------- Captured stdout teardown -----------------------------------------------------------------------------------------
2020-04-30T14:15:50.1588256150
========================================================================================= short test summary info ==========================================================================================
FAILED test_fim/test_basic_usage/test_basic_usage_new_dirs.py::test_new_directory[get_configuration1-tags_to_apply0] - TimeoutError
======================================================================================= 1 failed, 1 passed in 58.73s =======================================================================================
```

The proposed solution is to configure this test for performing a scan in a different frequency and the timeout to a greater value than the configured frequency.

--- a/tests/integration/test_fim/test_basic_usage/data/wazuh_conf_new_dirs.yaml
+++ b/tests/integration/test_fim/test_basic_usage/data/wazuh_conf_new_dirs.yaml
@@ -1,0 +1,18 @@
+---
+# conf 1
+- tags:
+  - ossec_conf
+  apply_to_modules:
+  - MODULE_NAME
+  sections:
+  - section: syscheck
+    elements:
+    - disabled:
+        value: 'no'
+    - frequency:
+        value: 30
+    - directories:
+        value: TEST_DIRECTORIES
+        attributes:
+        - check_all: 'yes'
+        - FIM_MODE

--- a/tests/integration/test_fim/test_basic_usage/data/wazuh_conf_new_dirs_win32.yaml
+++ b/tests/integration/test_fim/test_basic_usage/data/wazuh_conf_new_dirs_win32.yaml
@@ -1,0 +1,20 @@
+---
+# conf 1
+- tags:
+  - ossec_conf
+  apply_to_modules:
+  - MODULE_NAME
+  sections:
+  - section: syscheck
+    elements:
+    - disabled:
+        value: 'no'
+    - frequency:
+        value: 30
+    - windows_audit_interval:
+        value: WINDOWS_AUDIT_INTERVAL
+    - directories:
+        value: TEST_DIRECTORIES
+        attributes:
+        - check_all: 'yes'
+        - FIM_MODE

--- a/tests/integration/test_fim/test_basic_usage/test_basic_usage_new_dirs.py
+++ b/tests/integration/test_fim/test_basic_usage/test_basic_usage_new_dirs.py
@@ -11,7 +11,7 @@ import pytest
 
 from wazuh_testing import global_parameters
 from wazuh_testing.fim import detect_initial_scan
-from wazuh_testing.fim import generate_params, regular_file_cud, check_time_travel, callback_non_existing_monitored_dir
+from wazuh_testing.fim import generate_params, regular_file_cud, callback_non_existing_monitored_dir
 from wazuh_testing.tools import PREFIX, LOG_FILE_PATH
 from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
 from wazuh_testing.tools.monitoring import FileMonitor
@@ -26,7 +26,8 @@ test_directories = []
 directory_str = os.path.join(PREFIX, 'testdir1')
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 configurations_path = os.path.join(test_data_path,
-                                   'wazuh_conf.yaml' if sys.platform != 'win32' else 'wazuh_conf_win32.yaml')
+                                   'wazuh_conf_new_dirs.yaml' if sys.platform != 'win32'
+                                   else 'wazuh_conf_new_dirs_win32.yaml')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
 
 # Configurations
@@ -85,8 +86,6 @@ def test_new_directory(tags_to_apply, get_configuration, configure_environment, 
         regular_file_cud(directory_str, wazuh_log_monitor, file_list=['file1', 'file2', 'file3'],
                          min_timeout=global_parameters.default_timeout, triggers_event=False)
 
-        # Travel to the future to start next scheduled scan
-        check_time_travel(True)
         detect_initial_scan(wazuh_log_monitor)
     else:
         # Wait for syscheck to realize the directories don't exist
@@ -97,4 +96,4 @@ def test_new_directory(tags_to_apply, get_configuration, configure_environment, 
 
     # Assert that events of new CUD actions are raised after next scheduled scan
     regular_file_cud(directory_str, wazuh_log_monitor, file_list=['file4', 'file5', 'file6'],
-                     min_timeout=global_parameters.default_timeout, triggers_event=True)
+                     min_timeout=40, triggers_event=True)

